client/src/shared/components/Text-field/text-field.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface Props {
+  label: string;
+  placeholder: string;
+  width: number;
+  height: number;
+  fontsize: string;
+  resizable: boolean;
+}
+
+const Textfield: React.FC<Props> = ({ label, placeholder, width, height, fontsize, resizable }: Props) => {
+  return (
+          <div className="p-5 h-auto w-auto">
+            <div>
+              <label>{label} </label>
+            </div>
+            <textarea
+            rows = {height}
+            cols = {width}
+              className={`${
+                resizable ? 'resize' : 'resize:none'
+              } border border-black rounded-lg rows`}
+              placeholder={`${placeholder}`}
```

Comment:
- do not use backticks if the appended content is just one.
```suggestion
              placeholder={placeholder}
```